zcutil/fetch-pyarams.py

```diff
@@ -0,0 +1,296 @@
+# /************************************************************************
+#  File: fetch-pyarams.py
```

Comment:
Is the file supposed to be named `fetch-pyarams.py` or `fetch-params.py`? I see different instances of this across the file.

Reply:
Correct.